dashboard/lib/tasks/annotate_rb.rake

```diff
@@ -0,0 +1,8 @@
+# This rake task was added by annotate_rb gem.
+
+# Can set `ANNOTATERB_SKIP_ON_DB_TASKS` to be anything to skip this
+if Rails.env.development? && ENV["ANNOTATERB_SKIP_ON_DB_TASKS"].nil?
```

Comment:
This ENV option was added automatically by the `annotate_rb:install` script I ran [as recommended by the migration guide](https://github.com/drwl/annotaterb/blob/main/MIGRATION_GUIDE.md#automatic-annotations-after-running-database-migration-commands). I'm not entirely sure we want it, though; what do folks think about just doing this instead?

```suggestion
if Rails.env.development?
```